helpers/utils

```diff
@@ -167,7 +167,7 @@ ynh_setup_source() {
             || ynh_die --message="$out"
         # Check the control sum
         echo "${src_sum} ${src_filename}" | ${src_sumprg} --check --status \
-            || ynh_die --message="Corrupt source"
+            || ynh_die --message="Corrupt source for ${src_filename}: Expected ${src_sum} but got $(${src_sumprg} ${src_filename} | cut --delimiter=' ' --fields=1)."
```

Comment:
```suggestion
            || ynh_die --message="Corrupt source for ${src_filename}: Expected ${src_sum} but got $(${src_sumprg} ${src_filename} | cut --delimiter=' ' --fields=1) (size : $(du -hs ${src_filename} | cut --delimiter=' ' --fields=1))."
```

Proposing to also add the file size

Reply:
Ah yeah nice, with that it's easy to see if we actually got a 404 or other error code instead of the actual file.